lib/grape/endpoint.rb

```diff
@@ -188,6 +188,7 @@ def compile_path(prepared_path, anchor = true, requirements = {})
     end
 
     def call(env)
+      build_app
```

Comment:
On `YARV` - yes.
On 'JRuby' - theoretically `#build_stack` can be called multiple times but I don't see any side effect here( see the implementation of `#build_stack`); just extra an extra calculation, if many threads will try to set `@app`.
Anyway